.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "import/no-unresolved": "off",
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off"
+  },
```

Comment:
Kindly don't make changes to your linter file. Kindly remember that linter will help you follow best practices. You can use the right linter file from [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)

Reply:
Hello thank you for your kind review, here I did not change anything in linter.